cranelift/codegen/src/write.rs

```diff
@@ -530,7 +530,18 @@ pub fn write_operands(w: &mut dyn Write, dfg: &DataFlowGraph, inst: Inst) -> fmt
         FloatCondTrap {
             cond, arg, code, ..
         } => write!(w, " {} {}, {}", cond, arg, code),
+    }?;
+
+    let mut sep = "  ; ";
+    for &arg in dfg.inst_args(inst) {
+        if let ValueDef::Result(src, _) = dfg.value_def(arg) {
+            if let UnaryImm { imm, .. } = dfg[src] {
```

Comment:
What do you think about expanding this to `UnaryIeee{32, 64}` and `UnaryBool`?

Reply:
Okay, done! That still leaves `vconst` but that's more complicated since those constants aren't inline in the instruction. I don't feel like figuring out how that works right now since it was enough of a pain fixing the tests for these changes already. I'd be happy to review a PR on top of this one if somebody feels like taking that on though.

Reply:
I changed my mind: it's good enough to just print which element of the constant pool is in a vector-typed value, and that was easy to do. I think that might actually be more useful than printing giant vector literals. So now there's output for `vconst` as well.